frontend/src/Pages/Customer/ViewShops.js

```diff
@@ -0,0 +1,165 @@
+import axios from 'axios';
+import React from 'react';
+import { Col, Button, Card, Container, Form, ListGroup } from 'react-bootstrap';
+import { Link } from "react-router-dom";
+import '../../App.css';
+import LocationInput from '../LocationInput';
+
+const SEARCH_QUERY_URL = "https://speedy-anthem-217710.an.r.appspot.com/api/query/elastic";
+const BROWSE_QUERY_URL = "https://speedy-anthem-217710.an.r.appspot.com/api/browse/elastic";
```

Comment:
Consider putting these in `/src/routes.js`

Reply:
Done. Thank you!!! Left the viewCatalog one for now. Will change it when I add buyercatalog